tests/Domain/Insights/InvalidPhpCodeTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Domain\Insights;
+
+use NunoMaduro\PhpInsights\Domain\FileProcessors\FixerFileProcessor;
+use NunoMaduro\PhpInsights\Domain\Differ;
+use Symfony\Component\Finder\SplFileInfo;
+use Tests\TestCase;
+
+final class InvalidPhpCodeTest extends TestCase
+{
+    public function testNotFailingOnSemiColonAfterExtendClass(): void
+    {
+        $path = __DIR__ . '/Fixtures/InvalidPhpCode/SemiColonAfterExtendClass.php';
+
+        $file = new SplFileInfo($path, $path, $path);
+
+        $fixerFileProcessor = new FixerFileProcessor(new Differ());
+
+        $fixerFileProcessor->processFile($file);
+
+        self::assertTrue(true);
```

Comment:
:sweat_smile: 

You should try `$this->expectNotToPerformAssertions();` instead

Reply:
@Jibbarth My bad :D Thanks for the tip! Could you test new changes and give me a feedback? It's perfectly works for me now, but I want to be sure I didn't miss something.